app/models/cashflow.rb

```diff
@@ -0,0 +1,4 @@
+class Cashflow < ApplicationRecord
+  belongs_to :user
+  belongs_to :categories, class_name: 'Category', foreign_key: 'category_id'
```

Comment:
- [ ] Let's add model validations to avoid entering bad data.

Reply:
fixed

Reply:
well resolved! ✔️